sysutils/dcfldd/Portfile

```diff
@@ -1,29 +1,32 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
```

Comment:
You should make use of the github PG and use "github.setup"; that will simplify many things further down in the Portfile

Reply:
I had already used it, although it was further down in the `Portfile`. I moved it up closer to the github PG statement.